test/unit/article/article.spec.js

```diff
@@ -147,6 +148,60 @@ describe('Test for article', () => {
       expect(response.status).to.equal(400);
       expect(response.body.message).to.be.deep.equals('All fields are required');
     });
+
+    describe('tests for article ratings', () => {
```

Comment:
@marcdomain, great job. looking through your test I noticed that you are not testing for the value of `rating` that comes from the response object. Don't you think its a good idea to test for that since the `rating` is part of the response as shown on line 35 of the `ArticleRatingController`? Same apply

Reply:
The test script has been updated to cover for the ratings.
Thanks for the feedback.